core/src/OpenACC/Kokkos_OpenACC_Team.hpp

```diff
@@ -235,6 +235,8 @@ class TeamPolicyInternal<Kokkos::Experimental::OpenACC, Properties...>
   std::array<size_t, 2> m_thread_scratch_size;
   bool m_tune_team_size;
   bool m_tune_vector_length;
+  constexpr static const size_t default_team_size_max =
+      OpenACCTeamMember::DEFAULT_TEAM_SIZE_MAX;
   constexpr static const size_t default_team_size =
       OpenACCTeamMember::DEFAULT_TEAM_SIZE_REC;
```

Comment:
`OpenACCTeamMember::DEFAULT_TEAM_SIZE_{MAX,REC}` are defined as `int` and here we are casting them to `size_t` and implicitly casting them back to `int` when returning in the `team_size_{max,rec}` accessors or passing them to `init`.

Also the `const`-qualifier is superfluous here since it is `constexpr`

Reply:
Thank you for catching this; changed `size_t` to `int` and removed redundant `const` qualifier.